cli/configuration/index.md

```diff
@@ -0,0 +1,302 @@
+Rename done.
+```
+
+The directory name is fixed and the configuration added to the manifest:
+
+{: .image-popup}
+![Screenshot -- Fixed configuration directory](/cli/configuration/configurations-copy-2.jpg)
+
+## Configuration Rows
+
+The directory structure of configuration rows is the same as the configuration itself. The component configuration
+contains directory `rows` that includes a directory for each row. That directory contains `config.json`, 
+`description.md` and `meta.json`.
+
+Example of `meta.json`:
+```json
+{
+  "name": "share/cities2",
+  "isDisabled": false
+}
+```
+
+Example of a Google Drive Extractor configuration:
+
+{: .image-popup}
+![Screenshot -- A configuration rows directory example](/cli/configuration/directory-rows-example.jpg)
+
+## Transformations
+
+In addition to other configurations, the transformations directories contain `blocks` directory and in it a list of codes.
+Codes are stored in native files according to the type of transformation. I.e. Snowflake transformations store the codes
+in `.sql` files.
+
+Example of a Snowflake Transformation configuration:
+
+{: .image-popup}
+![Screenshot -- A transformation directory example](/cli/configuration/directory-transformation-example.jpg)
+
+## Variables
+
+[Variables](/transformations/variables/#variables) directory in addition to standard configuration layout contains
+directory `values`.
+
+Let's say you have these two variables in your transformation:
+
+{: .image-popup}
+![Screenshot -- Variables in the UI](/cli/configuration/variables-ui.jpg)
+
+When you [pull](/cli/commands/pull/) them to the local directory it will look like this:
+
+{: .image-popup}
+![Screenshot -- Configuration directory with the variables](/cli/configuration/variables-directory.jpg)
+
+Variables configuration in `variables/config.json`:
+
+```json
+{
+  "variables": [
+    {
+      "name": "state",
+      "type": "string"
+    },
+    {
+      "name": "city",
+      "type": "string"
+    }
+  ]
+}
+```
+
+Default values configuration in `variables/values/default/config.json`:
+
+```json
+{
+  "values": [
+    {
+      "name": "state",
+      "value": "NY"
+    },
+    {
+      "name": "city",
+      "value": "Boston"
+    }
+  ]
+}
+```
+
+## Shared Codes
+
+[Shared code](/transformations/variables/#shared-code) blocks are stored in the branch directory under `_shared` 
+subdirectory so that they can be reused between different configurations.
+
+If you create a shared code from your block:
+
+{: .image-popup}
+![Screenshot -- Shared code directory](/cli/configuration/shared-code-ui.jpg)
+
+It will move to the `_shared` directory:
+
+{: .image-popup}
+![Screenshot -- Shared code directory](/cli/configuration/shared-code-directory.jpg)
+
+And the code in the transformation file `blocks/001-block-1/001-join/code.sql` is changed to:
+
+{: .image-popup}
+![Screenshot -- Shared code code](/cli/configuration/shared-code-code.jpg)
+
+## Schedules
```

Comment:
Tu je potrebne doplnit sekciu pre orchestrator, ktory som pridaval adhoc (`phases`/`tasks`).
Screenshoty z UI by som velmi nedaval, kedze tam sa to bude este dost menit.